ui/app/pages/home/home.component.js

```diff
@@ -157,11 +161,49 @@ export default class Home extends PureComponent {
       </MultipleNotifications>
     )
   }
+  renderPopover = () => {
+    const { setConnectedStatusPopoverHasBeenShown } = this.props
+    const { t } = this.context
+    return (
+      <Popover
+        title={ t('whatsthis') }
+        onClose={() => setConnectedStatusPopoverHasBeenShown()}
+        className="home__connected-status-popover"
+        showTooltip
+        CustomBackground={({ onClose }) => {
+          return (
+            <div
+              className="home__connected-status-popover-bg-container"
+              onClick={onClose}
+            >
+              <div className="home__connected-status-popover-bg" />
+            </div>
+          )
+        }}
+      >
+        <main className="home__connect-status-text">
+          <div>{ t('metaMaskConnectStatusParagraphOne') }</div>
+          <div>{ t('metaMaskConnectStatusParagraphTwo') }</div>
+        </main>
+        <div className="home__connect-status-button-container">
```

Comment:
A `footer` prop is accepted by the Popover that applies many of these styles automatically

Reply:
Updated to use the footer prop in 593fdffa2